### ISSUE TITLE
GUI and rows with varying fields

### ISSUE THREAD

**Issue by mgreen27** [Velocidex/velociraptor#2294]
There appears to be some nuance to how the Velociraptor notbook gui processes rows with varying fields. 
The GUI api call is actually truncating the columns.

for example expanding out an Evtx EventData:
```
SELECT * FROM foreach(row=results, query={
    SELECT *,datetime,Computer,EventID,timestamp_desc,message  FROM EventData
})
```
may return varied row coumns based on the event log ids processed.

This impacts the displayed rows gui depending on the frist row in view. This seems to be a limitation on gui api querying the data and exporting to csv/other fields. The same query run in VQL on the commandline appears to work as expected.

A potential workaround solution could be a: write_jsonl() function
We may want to also consider gui api refactor.

![image](https://user-images.githubusercontent.com/13081800/205183466-f105af35-2aa1-4d3d-8bea-0f31714c1413.png)

**Comment by scudette** [comment]
There it actually more nuanced - when exporting the json or csv from the table widget users usually expect the table transformation to apply (i.e. the hidden columns should be removed, filtering applied etc).

This means that in practice the export mirrors the GUI exactly (this is what we actually want! ) 

So if the GUI can not represent varying columns because the tables are not properly formatted then the download json/csv can not represent these tables either.

While VQL itself does not care about varying columns in each row (since each row is represented as a json blob) it is possible to use the raw jsonl files or export the notebook to get those raw files as well.

**Comment by scudette** [comment]
Fixed by #2425
